core/error/error_macros.h

```diff
@@ -403,6 +403,13 @@ void _err_flush_stdout();
 	} else                                                                                             \
 		((void)0)
 
+#define ERR_FAIL_COND_STDOUT(m_cond)                                                                          \
+	if (unlikely(m_cond)) {                                                                            \
+		printf(FUNCTION_STR, __FILE__, __LINE__, "Condition \"" _STR(m_cond) "\" is true."); \
```

Comment:
This won't work you've just applied the same arguments as with the report function, you need to format it correctly

Reply:
Like so:
```suggestion
		fprintf(stderr, "ERROR: Condition \"%s\" is true.\n   at: %s (%s:%i)\n", FUNCTION_STR, __FILE__, __LINE__, _STR(m_cond)); \
```

Reply:
You reverted this